.github/workflows/cicd-main.yml

```diff
@@ -131,16 +131,16 @@ jobs:
         ### \'\'
   
   # L0: GPU unit tests
-  OPTIONAL_L0_Unit_Tests_GPU_ASR:
+  L0_Unit_Tests_GPU_ASR:
      needs: [cicd-test-container-setup]
```

Comment:
Please make sure to also re-add them to the final job `Nemo_CICD_Test` (into the `needs` list)

Reply:
Ok, thanks, I will do this! 
I'm currently trying to reproduce the issue on CI.
For RNN-T alignments, I do not see failures when running locally.
For some other tests, there are strange issues with serialization. If reproducible, I will try to fix it.

Reply:
Maybe there’s a cuda driver mismatch? If you cannot reproduce it locally?

Reply:
The problem is with the precision of calculations. I reduced the number of files. See details in Slack.